ovn-tester/tests/network_policy_cross.py

```diff
@@ -0,0 +1,38 @@
+from collections import namedtuple
```

Comment:
To be aligned with the existing network policy test case this file should be called `netpol_ns_cross.py`.